omnibus/config/software/python3.rb

```diff
@@ -1,21 +1,21 @@
 name "python3"
 
 if ohai["platform"] != "windows"
-  default_version "3.8.16"
+  default_version "3.9.16"
```

Comment:
nit - should we bump it to [3.9.17](https://docs.python.org/release/3.9.17/)?

Reply:
Hadn't noticed this. Thanks for the headsup, I'll bump it.